Task9.5.Unique_words.py

```diff
@@ -0,0 +1,45 @@
+from collections import defaultdict
+
+INPUT_FILE = 'Text1.txt'
+OUTPUT_FILE = 'words_and_its_frequency.txt'
+ALPHABET_UKR = 'абвгґдеєжзиійїклмнопрстуфхцчщьюя'
+
+
+def main():
+    words_list = create_words_list(INPUT_FILE)
+    dic_with_results = dic_with_words_and_its_frequency(words_list)
+    create_file_with_results(OUTPUT_FILE, dic_with_results)
+
+
+def create_words_list(name_file):
+    with open(name_file) as file:
+        text_from_file = file.read()
+        words_list = []
+        word = ''
+
+        for letter in text_from_file.lower():
+            if letter in ALPHABET_UKR:
+                word += letter
+            else:
+                if word != '':
+                    words_list.append(word)
```

Comment:
maybe we can already count this word, to avoid having another `for` loop in `dic_with_words_and_its_frequency`?